crates/turbopack-dev/src/react_refresh.rs

```diff
@@ -89,11 +89,12 @@ impl Issue for ReactRefreshResolvingIssue {
     }
 
     #[turbo_tasks::function]
-    fn description(&self) -> Vc<String> {
-        Vc::cell(
+    fn description(&self) -> Vc<StyledString> {
+        StyledString::String(
             "React Refresh will be disabled.\nTo enable React Refresh, install the \
              `react-refresh` and `@next/react-refresh-utils` modules."
```

Comment:
Code

Reply:
I would be nice to have a `styled!` macro that converts markdown to styled string. But that's probably a lot work...